typescript/packages/plugins/allora/src/allora.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Tool } from '@goat-sdk/core'
+import { createToolParameters } from '@goat-sdk/core'
+import { z } from 'zod'
+import { AlloraAPIClient } from './api'
+
+export class AlloraService {
+    private readonly client: AlloraAPIClient
+
+    constructor(apiKey: string) {
+        this.client = new AlloraAPIClient(apiKey)
+    }
+
+    @Tool({
+        description: 'Fetch a future price prediction for a crypto asset from Allora Network.  Specify 5 minutes from now `5m`, or 8 hours from now `8h`.',
+    })
+    async getPricePrediction(params: GetAlloraPricePredictionParams) {
```

Comment:
This probably will error when running. Tool methods expect a `paramaters` input not `params`

Reply:
Fixed